### PR TITLE
directly browse to select tomcat server or war package without clicking browse

### PR DESCRIPTION
**Pull request by SummerSun**

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -122,7 +122,8 @@ async function serverOpenConfig(tomcat: TomcatController, tomcatItem ?: TomcatSe
 
 async function createServer(tomcat: TomcatController): Promise<string> {
     const ui: VSCodeUI = new VSCodeUI();
-    const tomcatPath: string = await selectFolder(ui, Utility.localize('tomcatExt.selectinstllpath', 'Select Tomcat Installation Path'));
+    const tomcatPath: string = await ui.showFileFolderDialog(false, true, Utility.localize('tomcatExt.selectserver', 'Select Tomcat Server'));
```
Comment (andxu) [inline src/extension.ts]: Why to change 'Select Tomcat Installation Path' to 'Select Tomcat Server'

Comment (SummerSun) [reply src/extension.ts]: Previous behavior uses Select Tomcat Installation Path as hint but user have to do one more click the "Browse...":
click1

Update the behavior to open File explorer directly with hint to select folder to save user one button click effort:
click2

Comment (andxu) [reply src/extension.ts]: I don't think the phrase "select tomcat server" is proper for the folder selector, keep the original label:  "Select Tomcat Installation Path" or "Select Tomcat Directory" for the folder selector for the hint of a tomcat location/folder.

Comment (SummerSun) [reply src/extension.ts]: I will take the "Select Tomcat Directory".

**Pull request merged**